src/domain/forms/MultipageForm.tsx

```diff
@@ -59,7 +59,11 @@ interface Props {
    * Function that is called with a function that changes step and current step index,
    * and should validate the step and execute the given function if step is valid.
    */
-  stepChangeValidator?: (changeStep: () => void, stepIndex: number) => void;
+  stepChangeValidator?: (
+    changeStep: () => void,
+    stepIndex: number,
+    direction?: 'forward' | 'backward',
```

Comment:
If I've understood it correctly validation should happen when navigating to either direction. Now when navigating backward there is still the error if business ID is invalid.